app/controllers/links_controller.rb

```diff
@@ -38,6 +40,19 @@ def documentation_url(section = "", options = {})
     root_url + (section || '')
   end
 
+  # For new documentation at docs.theforeman.org
+  # options:
+  #   :section
+  #   :index    'foreman-el' (default), 'foreman-deb' or 'katello'
+  #   :chapter
+  def new_documentation_url(options)
+    version = SETTINGS[:version].short
```

Comment:
One thing to note is that in nightly this is `3.0` while the documentation URL will have `nightly` instead. For the current documentation we have a rewrite in place but AFAIK that isn't there for the new documentation.

Reply:
We have solved it in downstream with a config file in a known place, that overrides the default. We can ship with the config file set up accordingly.